jujugui/static/gui/src/app/components/expert-card/experts.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 module.exports = {
-  spicule: {
+  spiculecharms: {
```

Comment:
is there any code that needs to pick up this change?

Reply:
Nope, its just a map that gets checked when an entity is deployed.